pkg/apps/install.go

```diff
@@ -175,6 +175,12 @@ func (o *InstallOptions) DeleteApp(app string, alias string, releaseName string,
 		Items: make([]string, 0),
 	}
 
+	// Make sure that we use a temporary directory for GetApps even if we're not using a temporary directory for DeleteApps
```

Comment:
What does that mean? Thought we are going to use tmp directories for all of it now?

Reply:
That's an out-of-date comment - I'll remove.

Reply:
...and out of date code. Fixing. =)